.travis.yml

```diff
@@ -1,28 +1,71 @@
-dist: trusty
-sudo: required
+packagecloud: &packagecloud
```

Comment:
What’s the & do?

Reply:
This defines a variable of some sort in YAML, allowing us to substitute them later using the `<<: *packagecloud` syntax to avoid repeating all these key/value pairs.

http://yaml.org/type/merge.html